app/src/pages/portal/curation/ManageCuration.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <div>
+    <div class=" viz-u-mgup-sm utility-margin md-theme-default">
+      <div class="md-card-header contactus_radios md-card-header-flex">
+        <div class="md-card-header-text">
+          <div class="md-body-1">
+            Administrators can manage curations, like for example - delete, approve, and update curated samples status
```

Comment:
I'm not sure if this works as one sentence. Maybe 
"Administrators can manage curations, e.g., delete, approve, and update curated samples status."
or
"Administrators can manage curations. Accepted actions include delete, approve, and update curated samples status"